src/zac/core/forms.py

```diff
@@ -323,37 +323,95 @@ class SelectUsersReviewRequestForm(forms.Form):
         help_text=_("Select the advisors."),
     )
 
+    deadline = forms.DateField(
+        required=True,
+        label=_("Deadline"),
+        help_text=_("Select a date"),
+        input_formats=["%Y-%m-%d"],
+    )
+
 
 class BaseReviewRequestFormSet(BaseTaskFormSet):
-    def is_valid(self):
-        super().is_valid()
+    def deadlines_validation(self) -> bool:
+        """
+        Validate that deadlines per step are monotonic increasing
+        """
+        valid = True
+        deadline_old = date(1, 1, 1)
```

Comment:
```suggestion
        deadline_old = date.today()
```
Deadlines should be in the future imo, so we can use a realistic base-line with today's date?